### ISSUE TITLE
Javascript functions not working on eBay

### ISSUE THREAD

**Issue by Robin-Whittle** [MrAlex94/Waterfox#2048]
Hi, I am using G3.21. (64 bit) on Windows 7.  In the last day, a number of functions have stopped working on eBay.  

For instance, at: https://www.ebay.com/itm/265128213397 , clicking on the main picture should create a large version of it.  Clicking on the smaller pictures below it should make that one the main one. Clicking on the arrows should scroll through multiple small pictures.  Clicking on "Add to Watchlist" should lead to a pull-down list.  "Add to card" works.

I assume that Waterfox has not suddenly become broken, so I guess that eBay is not playing nice with particular browsers.

Thanks for Waterfox!

**Comment by Robin-Whittle** [comment]
This page, with an originally longer URL, was working fine, but then stopped working, in the same browser session: https://www.ebay.com.au/itm/254638659479 .    (https://www.ebay.com.au/itm/XT-XINTE-Digital-Toslink-Audio-to-Mini-3-5mm-Cable-Optical-Audio-Adapter-Cable/254638659479)  .  All the other eBay pages I have looked at recently did not work, including the same page from ebay.co.uk.

So this is an odd, flaky problem.  I am not expecting a fix from the Waterfox developers, but wanted to report it in case someone else observed it too.

**Comment by Robin-Whittle** [comment]
I have had no problems with the main picture as described for the past few weeks.   However, there is another intermittent, usually present, problem which I think has been going on for two months or so.

In my Watch List page: https://www.ebay.com.au/mye/myebay/WatchList there is a section near the top "All categories (32)".  To the right of this are a bunch of items such as "Drives, Storage & Blank Media".  This is supposed to run to a particular width where the items fade to white and no more appear.  This is what happens on Firefox Developer Edition and on Chrome (this is all Windows 7 64 bit Prof.).   However, most of the time, on Waterfox, there is no fading and the list stretches way off to the right.   This sets the width of the table which contains most of the items in this page and it is very wide, such as about 6000 pixels.   

This matters since at the bottom, where there are links [1] [2] [3] etc. to find other pages of results, which is in the middle of the table, horizontally, is about 3500 pixels to the right of the left of the whole page.   Now I know this I can see it, since I have a 4k monitor, but for the last two months or so I have puzzled at this problem of having no such set of links.

**Comment by MrAlex94** [comment]
Wouldn't surprise me if eBay A/B test features and that ends up breaking things for browser like Waterfox. Appreciate the report! But not much to be done on our side.

**Comment by Robin-Whittle** [comment]
I am not suggesting that this bug be re-opened, but it seems like a good place to report continuing troubles with websites treating Waterfox differently from Firefox in a disruptive way.   Today, the above problem with the extreme width of the Watchlist page was replaced by it rendering in a bizarre format, with no left set of menus for Purchase History and the like.

![eBay-homepage-Waterfox](https://user-images.githubusercontent.com/13086050/130318069-523a4be8-66b9-42ef-b35b-59ee3fd7e595.png)

I am running G3.2.4.1 (64-bit) on Windows 7.

**Comment by Robin-Whittle** [comment]
I installed Waterfox using "Waterfox G3.2.5 Setup.exe" on another computer, and the page was not scrambled - though it had the same excessive width problem I reported on 2021-05-12.  I did the same on my main computer where this problem occurred and the scrambling problem was fixed.  The new installation worked with my current profiles.   So this scrambling problem may not be related to to eBay, but to something odd in Waterfox updating itself since I last installed it with "Waterfox G3.1.0 Setup.exe on 2021-02-11.   I mentioned this to eBay help via chat and pointed them to this page.  The fellow there said he would pass this on to his technical team.
